src/screens/SummaryScreen.js

```diff
@@ -53,11 +53,11 @@ const SummaryScreen = ({
         <Demographics />
         <View style={styles.resourceTypeContainer}>
           {Object.entries(resourceIdsGroupedByType).map(
-            ([resourceType, resourceIds]) => (
+            ([resourceType]) => (
               <ResourceTypeRow
                 key={resourceType}
                 resourceType={resourceType}
-                resourceIds={resourceIds}
+                total="TBD"
```

Comment:
With the introduction of `subTypes`, the data shape of `resourceIdsGroupedByType` has changed and you can no longer call on `resourceIdsGroupedByType[resourceType].size. I've put the `"TBD"` string here for now and it can be updated after timeline state is implemented.